packages/plugin-multi-tenant/src/utilities/addFilterOptionsToFields.ts

```diff
@@ -170,42 +170,54 @@ function addFilter<ConfigType = unknown>({
   // User specified filter
   const originalFilter = field.filterOptions
   field.filterOptions = async (args) => {
-    const originalFilterResult =
-      typeof originalFilter === 'function' ? await originalFilter(args) : (originalFilter ?? true)
+    const getResult = async () => {
```

Comment:
The diff from line 173 - 211 is just wrapping the original code inside `getResult`.